scopes/lanes/ui/overview/menu/lanes-overview-menu.tsx

```diff
@@ -50,20 +52,26 @@ export function LanesOverviewMenu({ navigationSlot, widgetSlot, className }: Lan
     </div>
   );
 }
-
 function MenuNav({ navigationSlot }: { navigationSlot: LanesOrderedNavigationSlot }) {
-  const plugins = navigationSlot.toArray().sort(sortFn);
+  const plugins = flatten(
+    navigationSlot.toArray().map(([id, values]) => {
+      const flattenedValues = flatten(values).map((value) => ({ ...value, id }));
+      return flattenedValues;
+    })
+  ).sort(sortFn);
```

Comment:
seems like a lot to extract the data. maybe we can pass the data in a better way? so it would be so hard to extract